Content.Server/GameObjects/Components/Mobs/Body/Limb.cs

```diff
@@ -0,0 +1,322 @@
+        int CurrentHealth;
+        IEntity Owner;
+        BodyTemplate BodyOwner;
+        float BloodChange = 0f;
+        string PrototypeEntity = "";
+        bool childOrganDamage; //limbs that are used for deattaching, they'll get deleted if parent is dropped. (arms, legs)
+        bool directOrganDamage;
+        List<string> _organProt = new List<string>();
+        /// <summary>
+        /// Additional targets like Mouth and Eyes.
+        /// </summary>
+        Dictionary<string, string> _addTargetKeys = new Dictionary<string, string>();
+        List<AdditionalTarget> _addTargets = new List<AdditionalTarget>();
+        List<LimbStatus> Statuses = new List<LimbStatus>();
+
+
+        string IIndexedPrototype.ID => Id;
+
+        void IPrototype.LoadFrom(YamlMappingNode mapping)
+        {
+            var obj = YamlObjectSerializer.NewReader(mapping);
+            obj.DataField(ref Id, "id", "");
+            obj.DataField(ref Name, "name", "");
+            obj.DataField(ref MaxHealth, "health", 0);
+            obj.DataField(ref TargetKey, "target", "");
+            obj.DataField(ref TexturePath, "dollIcon", "");
+            obj.DataField(ref PrototypeEntity, "prototype", "");
+            obj.DataField(ref Parent, "parent", "");
+            obj.DataField(ref childOrganDamage, "childOrganDamage", false);
+            obj.DataField(ref directOrganDamage, "directOrganDamage", false);
+
+            if (mapping.TryGetNode<YamlSequenceNode>("additionalTargets", out var targetNodes))
+            {
+                foreach (var targetMap in targetNodes.Cast<YamlMappingNode>())
+                {
+                    ReadAddTargetPrototype(targetMap).ToList().ForEach(x => _addTargetKeys.Add(x.Key, x.Value));
+                }
+            }
+            if (mapping.TryGetNode<YamlSequenceNode>("organs", out var organNodes))
+            {
+                _organProt = new List<string>();
+                foreach (var prot in organNodes.Cast<YamlMappingNode>())
+                {
+                    _organProt.Add(prot.GetNode("map").AsString());
+                }
+            }
+        }
+
+        public void Initialize(IEntity owner, BodyTemplate body)
+        {
+            PrototypeManager = IoCManager.Resolve<IPrototypeManager>();
+            reflectionManager = IoCManager.Resolve<IReflectionManager>();
+            CurrentHealth = MaxHealth;
+            Statuses = new List<LimbStatus>();
+            Owner = owner;
+            BodyOwner = body;
+            _seed = new Random(DateTime.Now.GetHashCode());
+            Organs = new List<Organ>();
+            foreach (var key in _organProt)
+            {
+                var prot = PrototypeManager.Index<OrganPrototype>(key);
+                var organ = prot.Create();
+                organ.Initialize(owner, body);
+                Organs.Add(organ);
+            }
+            if (reflectionManager.TryParseEnumReference(TargetKey, out var @enum))
```

Comment:
`TryParseEnumReference` is not intended for parsing SPECIFIC enums. It's intended to parse flexible keys. You should just deserialize to this enum directly.